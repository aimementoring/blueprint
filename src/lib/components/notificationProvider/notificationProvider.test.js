import React from 'react';
import renderer from 'react-test-renderer';
import NotificationProvider from './notificationProvider.js';

describe('NotificationProvider', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <NotificationProvider>
          <div>My App</div>
        </NotificationProvider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
