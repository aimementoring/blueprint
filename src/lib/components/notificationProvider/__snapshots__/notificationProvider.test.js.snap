// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationProvider renders properly 1`] = `
<div>
  My App
</div>
`;
